examples/address_verification.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'smile-identity-core'
+# See https://docs.usesmileid.com/products/for-individuals-kyc/address-verification for
+# more information on address verification
+
+# Initialize
+partner_id = '' # login to the Smile Identity portal to view your partner id
+api_key = '' # copy your API key from the Smile Identity portal
+sid_server = '0' # Use 0 for the sandbox server, use 1 for production server
+
+connection = SmileIdentityCore::AddressVerification.new(partner_id, api_key, sid_server)
+
+request_params = {
```

Comment:
I would encourage to use two examples for `ZA` and `NG` as they require different fields. You could create an array of example cases and loop through them making requests.

Reply:
Done. Thank you!